### PR TITLE
Add more styling to SAP registration form (and some bug fixes)

### PR DESCRIPTION
**Pull request by polybuildr**

### CODE REVIEW

**Comment by erilyth** [comment]
This looks great :D :+1: , A lot better without the table.

File: src/app/views/sap/register.php [hunk 1]
```diff
@@ -1,54 +1,107 @@
...
-<?php
-?>
 <html lang="en">
 <head>
     <meta charset="utf-8">
     <title>Student Ambassador Program · Felicity ʼ16</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="<?= base_url() ?>static/styles/vendor/thoda.min.css">
-    <link rel="stylesheet" href="<?= base_url() ?>static/styles/sap_register.css">
+    <link rel="stylesheet" href="https://felicity.iiit.ac.in/css/vendor/thoda.min.css">
```
Comment (polybuildr) [inline src/app/views/sap/register.php:7]: Yes, this is very bad. But the deploy was so painful, I ended up doing this. Needs fixing.

Comment (erilyth) [inline src/app/views/sap/register.php:7]: Hmm, should this be opened up as another issue?

Comment (polybuildr) [inline src/app/views/sap/register.php:7]: Yeah, it should.

**Pull request merged**